doc/02-installation.md

```diff
@@ -151,7 +151,7 @@ yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.r
 
 ```bash
 rpm --import https://packages.icinga.com/icinga.key
-dnf install https://packages.icinga.com/fedora/icinga-rpm-release-$(. /etc/os-release; echo "$VERSION_ID")-latest.noarch.rpm
+dnf config-manager --add-repo https://packages.icinga.com/fedora/$(. /etc/os-release; echo "$VERSION_ID")/release
```

Comment:
> No such command: config-manager. Please use /usr/bin/dnf --help
> It could be a DNF plugin command, try: "dnf install 'dnf-command(config-manager)'"

At least the output is helpful to fix the error.

Reply:
Not exactly sure what the point of requesting a review from me is here. The instructions should work and in the Debian instructions for example, wget is also installed explicitly.